docs/source/mb_specification.rst

```diff
@@ -22,26 +22,33 @@ A MONAI Bundle is defined primarily as a directory with a set of specifically na
   ┃  ┗━ metadata.json
   ┣━ models
   ┃  ┣━ model.pt
-  ┃  ┗━ model.ts
+  ┃  ┣━ model.ts
+  ┃  ┗━ model.onnx
   ┗━ docs
      ┣━ README.md
      ┗━ license.txt
 
 
-These files mostly are required to be present with the given names for the directory to define a valid bundle:
+The following files are **required** to be present with the given filenames for the directory to define a valid bundle:
 
 * **metadata.json**: metadata information in JSON format relating to the type of model, definition of input and output tensors, versions of the model and used software, and other information described below.
 * **model.pt**: the state dictionary of a saved model, the information to instantiate the model must be found in the metadata file.
+
+The following files are optional but must have these names in the directory given above:
```

Comment:
I think this is the discussion in Yiheng's previous PR, @wyli do you also think we should force the file names here?

Thanks.

Reply:
Which pr are you referring to?

Reply:
Hi @wyli ,

I mean this question for you: https://github.com/Project-MONAI/MONAI/pull/4114#discussion_r854211411.

Thanks.

Reply:
Ok I think we could have a primary 'model.pt', and optionally have some additional 'model_*.pt' files here

Reply:
The important thing from my perspective is that having fixed filenames means we can rely on that assumption in other code which consumes bundles, ie. the deploy operator. It's easier if we know the files `model.pt` and `metadata.json` are always there. Bundles also store only one model because it's so much easier to provide the metadata for one network and also because that's what Torchscript supports in its format, but having other `.pt` files for alternative weights or other data is perfectly fine with any other filename the user wants. I can discuss other optional files in more detail if that would help in the specification.